run-arch64.sh

```diff
@@ -0,0 +1,33 @@
+clear
```

Comment:
Instead of duplicating the existing `run.sh`, what if you just store the architecture to a variable and adjust the commands? E.g.:

```
ARCH=$(uname -m) 
chmod +x ./pppwn-${ARCH}
./pppwn-${ARCH} --interface $interface ...
```
If you don't want to rename the executable (arch64 to aarch64), you can use something like:
```
  ARCH=$(uname -m)
  case $ARCH in
    armv7*) ARCH="armv7";;
    aarch64) ARCH="arm64";;
  esac
```